website/src/Components/PageComponents/TournamentsList.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import Tournament from './Tournament'
+
+class TournamentsList extends Component{
+	//will take in a js objects created from server data and render them in a table
+	constructor(){
+		super();
+		this.state={
+			tournaments: [
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			],
+		}
+	}
+
+	renderTournaments(){
+		const tournaments = this.state.tournaments;
+		const listItems = tournaments.map((tourney) =>
+  		<Tournament id={tourney.id} description={tourney.description} creatorName={tourney.creatorName} address={tourney.address} />
+		);
+		return (
+			<table className='tourneyList'>{listItems}</table>
```

Comment:
Be careful when using default HTML options like table. Change this to a div as it is not guaranteed for each Tournament to be a <tr>